src/modules/auth/helper/fabricWallet.ts

```diff
@@ -73,33 +99,36 @@ export class FabricWallet {
 
     const adminUser = await fetchAdminUserFromId(adminId, FabricWallet.wallet);
 
+    const convertedObjects = convertStringToObjects(creds.certMetadata);
+
+    const attributesObj = [
+      {
```

Comment:
I think adding a comment here regarding the reason for `hf.Registrar` attributes being incorporated in the user certs will be super helpful for future references.

Reply:
Good point